activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -249,11 +251,51 @@ def #{unsafe_method}!(*args)              # def capitalize!(*args)
       end
     end
 
+    UNSAFE_STRING_METHODS_WITH_BACKREF.each do |unsafe_method|
+      if unsafe_method.respond_to?(unsafe_method)
+        class_eval <<-EOT, __FILE__, __LINE__ + 1
+          def #{unsafe_method}(*args, &block)             # def gsub(*args, &block)
+            if block                                      #   if block
+              to_str.#{unsafe_method}(*args) { |*params|  #     to_str.gsub(*args) { |*params|  
```

Comment:
Can you address the trailing spaces?

```suggestion
              to_str.#{unsafe_method}(*args) { |*params|  #     to_str.gsub(*args) { |*params|
```

![image](https://user-images.githubusercontent.com/12642/48205592-cf35ed80-e3af-11e8-92cc-972712b9646f.png)

https://codeclimate.com/github/rails/rails/pull/34405

Reply:
Sorry, I've fixed it.